Extension/src/LanguageServer/extension.ts

```diff
@@ -293,9 +294,38 @@ function realActivation(): void {
     updateLanguageConfigurations();
 
     reportMacCrashes();
-
+    
     const settings: CppSettings = new CppSettings(clients.ActiveClient.RootUri);
-    if (settings.updateChannel === 'Insiders') {
+    let suggestInsider: PersistentState<boolean> = new PersistentState<boolean>("CPP.suggestInsider", true);
+    
+    if (suggestInsider.Value && settings.updateChannel === 'Default') {
+        getTargetBuildInfo("Insiders").then(buildInfo => {
+            if (!buildInfo) {
+                return;
+            }
+            const message: string = `The insider version (${buildInfo.name}) is available, whether to switch to the insider channel?`;
```

Comment:
The Insiders version...switch to the Insiders channel?.

Reply:
Also, ", whether to" doesn't seem like the correct wording. Can you remove that, i.e. "is available. Switch to...".

Reply:
Yes, you are right, I made a change.

Reply:
I don't see the change to the `The Insiders version...` string.